docs/changelogs/v0.20.md

```diff
@@ -13,6 +13,18 @@
 
 ### 🔦 Highlights
 
+#### New gateway powers
+@lidel
```

Comment:
"we rewrote everything, but hopefully you won't notice" :trollface: 

jk, notes to self:

- [ ] check history for `boxo/gateway` and mention user-facing fixes
- [ ] mention work from  https://github.com/ipfs/boxo/pull/176, probably enough to point at  IPFSBackend interface
  - [ ] explain why we removed some metrics, and note which are new and experimental, ask for feedback on metrics/tracing 
- [ ] mention https://github.com/ipfs/gateway-conformance